### PR TITLE
[show techsupport] update the command list according to image changes

### PR DESCRIPTION
**Pull request by yxieca**
Summary:
Fixes # (issue)

### Type of change

- [x] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 201911

#### What is the motivation for this PR?
show tech support command test case is failing.

The change was from: https://github.com/Azure/sonic-utilities/pull/1723

#### How did you do it?
Update the command format

#### How did you verify/test it?
Run show_techsupport test.

Signed-off-by: Ying Xie <ying.xie@microsoft.com>

### CODE REVIEW

File: tests/show_techsupport/tech_support_cmds.py [hunk 1]
```diff
@@ -117,7 +117,7 @@
     "docker stats --no-stream",
     "docker ps -a",
     "docker top pmon",
-    "docker exec -it lldp{} lldpcli show statistics",
+    "docker exec lldp{} lldpcli show statistics",
```
Comment (neethajohn) [inline tests/show_techsupport/tech_support_cmds.py:120]: Can you update the description with the associated PR number? Also will this work for 201911?

Comment (yxieca) [reply tests/show_techsupport/tech_support_cmds.py:120]: Updated the comments. 201911 will fail this test unless using 201911 sonic-mgmt branch to test.

Comment (neethajohn) [reply tests/show_techsupport/tech_support_cmds.py:120]: Should we make this backward compatible?

**Review by neethajohn — APPROVED** [review]

File: tests/show_techsupport/tech_support_cmds.py (hunk 1 above)
Comment (yxieca) [reply tests/show_techsupport/tech_support_cmds.py:120]: sure. added.

**Review by neethajohn — APPROVED** [review]

**Pull request merged**